### ISSUE TITLE
Example is broken

### ISSUE THREAD

**Issue by evelant** [thheller/fulcro-expo#1]
Looks like this example is broken now for whatever reason. Everything compiles but running the app on a simulator errors with:

```
undefined is not an object (evaluating 'goog.LOCALE.substring')
```

**Comment by evelant** [comment]
Thanks for the update. It appears it still isn't working. Shadow-cljs 2.8.24 can't be found.

shadow-cljs - dependency update failed - Could not find artifact thheller:shadow-cljs:jar:aot:2.8.24 in central (https://repo1.maven.org/maven2/)

**Comment by thheller** [comment]
Hmm try again. The upload was aborted for some reason but it should be working now.

https://clojars.org/thheller/shadow-cljs

**Comment by evelant** [comment]
All the deps load correctly now thanks. The example starts now but immediately crashes with "console.error there was a problem sending log messages to your development environment" however that problem may be on my end.

edit: the logging  error seems  to be a bug in expo or react native https://forums.expo.io/t/updated-to-expo-sdk-v32-0-0-and-get-there-was-a-problem-sending-log-messages-to-your-development-environment-for-every-console-call-made/19377

Just out of curiosity for a newbie to clj/cljs  what was the source of this issue?

**Comment by thheller** [comment]
I don't know about the logging thing but I experience something similar before where the whole `expo` app would crash on certain `console.log` calls. Seems a bit brittle overall but isn't related to `shadow-cljs`, just gotta be careful what to send to the `console`.

> undefined is not an object (evaluating 'goog.LOCALE.substring')

This was caused by a recent change I made in `shadow-cljs` which fixed a different issue but in turn broke `goog.define("goog.LOCALE", "en");` which the Closure Library does on load for the `:npm-module` target (which the `:react-native` target uses during development). The define would end up in the wrong scope so `goog.LOCALE` ended up `undefined`.
